frontend/src/pages/AllChatsPage.jsx

```diff
@@ -45,7 +79,11 @@ export default function AllChatsPage() {
     return (
         <div className="AllChatsPage FullHeight LightBackground">
             <ChatInboxHeader />
-            <ChatInbox chats={matches} users={matchedUsers} />
+            <ChatInbox chats={unblockedMatches} users={matchedUsers} />
+
+            {/* TODO: Make this a collapsable element & hide it if no blockedMatches are present */}
+            <h1>Hidden Conversations</h1>
```

Comment:
Can we only show the hidden conversations header if there are any users that are blocked? It won't make much sense if there are no hidden conversations.